Update ping.md (#1728)
redis/redis-doc: commands/ping.md

@return

> @simple-string-reply, if no argument is provided
> 
> @bulk-string-reply, copy of the argument if provided

Reviewer: ```suggestion
@simple-string-reply, and specifically `PONG`, when no argument is provided.

@bulk-string-reply the argument provided, when applicable.
```
